ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main__4.15-nightly.yaml

```diff
@@ -168,6 +168,48 @@ tests:
     - chain: redhat-chaos-hog-scenarios
     workflow: redhat-chaos-installer-azure-aro-upi
   timeout: 5h0m0s
+- always_run: false
+  as: krkn-hub-tests-sdn2ovn
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      CHURN: "true"
+      COMPUTE_NODE_REPLICAS: "6"
+      CONTROL_PLANE_INSTANCE_TYPE: m5.2xlarge
+      EXTRA_FLAGS: --churn-duration=90m --timeout=5h --churn-percent 25
+      GC: "false"
+      IF_CREATE_WORKLOAD_NODE: "true"
+      ITERATION_MULTIPLIER_ENV: "11"
+      ITERATIONS: "54"
+      OVERRIDE_OPENSHIFT_SDN_DEPRECATION: "true"
+      PROFILE_TYPE: reporting
+      TELEMETRY_GROUP: prow-ocp-4.15
+      ZONES_COUNT: "3"
+    observers:
+      enable:
+      - redhat-chaos-cerberus
+      - openshift-qe-cluster-density-v2-observer
+    post:
+    - ref: openshift-qe-kube-burner-index
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-conf-aws
+    - ref: sdn-conf
+    - chain: ipi-install
+    - chain: create-infra-move-ingress-monitoring-registry
+    test:
+    - chain: openshift-qe-cluster-density-v2
```

Comment:
this line and line 193 should be used one or the other. They perform the same action/workload. you need to remove one of them

Reply:
Ok, remove this one and keep line 193, thanks!